hack/update-3generated-crd-code.sh

```diff
@@ -33,9 +33,11 @@ fi
 # get code-generation tools (for now keep in GOPATH since they're not fully modules-compatible yet)
 mkdir -p ${GOPATH}/src/k8s.io
 pushd ${GOPATH}/src/k8s.io
-git clone -b v0.22.2 https://github.com/kubernetes/code-generator
+#git clone -b v0.22.2 https://github.com/kubernetes/code-generator
 popd
 
+export GOPROXY=https://goproxy.io,direct
```

Comment:
Does everyone need this? If not, you could `GOPROXY=https://goproxy.io,direct ./hack/update-3generated-crd-code.sh` instead of making this change for everyone.

Reply:
Already updated.